dbos/cli/cli.py

```diff
@@ -347,29 +382,31 @@ def list(
 
 @workflow.command(help="Retrieve the status of a workflow")
 def get(
+    ctx: typer.Context,
```

Comment:
Three problems with doing it this way:

1. Not obvious how it actually works
2. It breaks the help text, the new option isn't displayed
3. If you don't pass in anything, you get a "`dbos-config.yaml` not found" error, which is not helpful.

Reply:
![Screenshot 2025-04-22 at 15 42 25](https://github.com/user-attachments/assets/c9ceb702-a1ca-469c-932e-8235bea03014)

I've been thinking about adding some debug info for where the database url comes from, with a verbose mode, but the challenge is that it's a pain to handle correctly with JSON output.

We can tolerate this error -- unlikely if you come from any of the starters that provide a config file  -- which will disappear in v1

Reply:
Okay sure but 
![image](https://github.com/user-attachments/assets/36464a6e-27e8-43c9-abea-98b037f99bc0)

Reply:
This is not obvious behavior: 

![image](https://github.com/user-attachments/assets/7a1631f4-c4d6-4c09-ba5e-9f8ad342baf1)

Reply:
yeah agreed, I'll make it an option in each of the subcommand.